### ISSUE TITLE
Text Localization

### ISSUE THREAD

**Issue by Sean343** [googleads/googleads-consent-sdk-android#14]
Hi,

Is it possible to localise the text so that it supports multiple languages? I have a number of users from non-english speaking countries.

**Comment by MachinHere** [comment]
Hello, any update on it? I think this is one of the most important case in this project right now - if it is intendend to inform all EU users appropriately they should be given explenation in their language.

**Comment by dzungpv** [comment]
Edit the lib yourself, create html for each language see this line https://github.com/googleads/googleads-consent-sdk-android/blob/master/consent-library/src/main/java/com/google/ads/consent/ConsentForm.java#L271

**Comment by iordachiv** [comment]
I think that native library/sdk should display also native dialog and not render html (with javascript and so one) instead.

**Comment by Diranus** [comment]
Idea for a quick solution:
Probably it would be useful to exclude the path of the consentform.html into a member variable which can be changed by setConsentFormUrl() or something like this to get multilanguage support without hacking the lib itself (I hate that! It breaks the ability to update). 
Then we can have multiple variants of consentform.html in the asset folder.

**Comment by dzungpv** [comment]
I think a simple edit can support other language:
```
String countryiso3 = Resources.getSystem().getConfiguration().locale.getISO3Country();
this.webView.loadUrl("file:///android_asset/consentform_"+"countryiso3"+".html");
```
And then translate the html form for each language, put it in your asset folder and you are done

**Comment by codenia** [comment]
Here are 10 languages for which I have ordered a translation. The texts were translated by a translation team. I cannot guarantee the correctness of the translation.

[consentform.zip](https://github.com/googleads/googleads-consent-sdk-android/files/2094036/consentform.zip)

**Comment by witoldgoralski** [comment]
Thank you for sharing. I think that Chinese, Japanese, Korean and Russian should not be provided with library. As far as I know only for EEA (European Economic Area) countries this consent should be displayed. European Economic Area countries: https://europa.eu/european-union/about-eu/countries_en#tab-0-0

**Comment by codenia** [comment]
Yes, but in Europe there are many Chinese people who have set Chinese language on their smartphone or many Russians who have set Russian language. (And so on)

**Comment by witoldgoralski** [comment]
It make sense for me. But it is bad news.

**Comment by vostreltsov** [comment]
I guess there is a bit better way to localize the form, without editing the google library's source code. We can access the private webView via reflection and do something like this (based on the codenia's post above):

```
public static void localizeConsentForm(ConsentForm form) {
    try {
        Field f = form.getClass().getDeclaredField("webView");
        f.setAccessible(true);
        WebView webView = (WebView) f.get(form);
        webView.loadUrl("file:///android_asset/consentform_ru.html");
    } catch (Exception e) {
    }
}

```

**Comment by vostreltsov** [comment]
Another way to localize this form is to execute javascript after the webview loads:
```
webView.loadUrl("javascript:function localize(){document.getElementsByClassName('head_intro')[0].innerHTML = 'localized string goes here'}; localize();");
```
and so on, for all elements. This should be called from the `onConsentFormLoaded()` listener.

You will not have to change the library source code. You just add your localized stuff in `strings.xml`, and put it to the js calls. No need to copy-paste html forms.

**Comment by vrerabek** [comment]
I am using #86 as temporary solution with my own consentform.html files and it's working well.
